zerver/views/message_edit.py

```diff
@@ -114,10 +114,10 @@ def update_message_backend(
     content: Optional[str] = REQ(default=None),
 ) -> HttpResponse:
     if not user_profile.realm.allow_message_editing:
-        return json_error(_("Your organization has turned off message editing"))
+        raise JsonableError(_("Your organization has turned off message editing"))
```

Comment:
I don't think we would like to throw errors in API calls.

Reply:
Then would `return json_error` be a better alternative here? I found that other functions in `zerver/lib/actions.py` do raise exceptions, perhaps it's reasonable to do so in this context.

Reply:
Yeah, that's true in actions we throw errors but the api client never faces an error, we would always want the api clients to get a proper response for the error thrown.